backend/molgenis-emx2-sql-it/src/test/java/org/molgenis/emx2/sql/TestGrantRolesToUsers.java

```diff
@@ -195,4 +196,19 @@ public void testRole() {
       database.clearActiveUser();
     }
   }
+
+  @Test
+  public void testCaseSensitiveSchemaNames() {
```

Comment:
Suggest having use tests for create, drop, .... and then adding assertion for case sensitive .  A integration test can then check is UserA can create , add .. some find ScHeMaA and not in schemaA

Reply:
tests added